notebooks/demo2/teacher_student_inference.ipynb

```diff
@@ -0,0 +1,4730 @@
+{
```

Comment:
<p>Would it be easier to compare across notebooks if we keep the data points and pages constant? So, we define what an average pdf size is, based on the data, Let's say 157 pages and 387 data points. Now, if we keep that constant we can make statements like model m takes t1 seconds and model n takes t2 for inferring with the average pdf. </p><p>When we take the sample size of 35, we assume that the time per data point for the 35 pdfs would reflect the mean for 145 files (given standard deviation is low, it's a fair assumption) But since we already know the average pages and no. of data points for the 145 files, we could use that value for our results even though the time per data point is averaged over 35 files. In other words, we can remove the cells here that calculate number of pages and total number of points and just get the time per data point and use 158 and 387 as a standard.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/248/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>